### PR TITLE
feat(dmmf): add full index information

### PR DESCRIPTION
**Pull request by aqrln**
Add complete index information to the `datamodel` section of the DMMF.

### CODE REVIEW

**Review by Druue — APPROVED** [review]
Nice!

File: prisma-fmt/src/get_datamodel.rs [hunk 1]
```diff
@@ -190,7 +190,50 @@ mod tests {
                   "isGenerated": false
                 }
               ],
-              "types": []
+              "types": [],
+              "indexes": [
```
Comment (SevInf) [inline prisma-fmt/src/get_datamodel.rs:194]: Would be nice to have a snapshot with `algorithm` and `clustered` properties too.

Comment (SevInf) [reply prisma-fmt/src/get_datamodel.rs:194]: And other properties of `IndexField` too

Comment (aqrln) [reply prisma-fmt/src/get_datamodel.rs:194]: There are actually! But in `dmmf` tests, not `prisma-fmt`.

Comment (aqrln) [reply prisma-fmt/src/get_datamodel.rs:194]: See the new tests in `query-engine/dmmf/test_files`.

Comment (aqrln) [reply prisma-fmt/src/get_datamodel.rs:194]: I can also add something here as well if you'd prefer.

**Pull request merged**